sdk/eventgrid/README.md

```diff
@@ -32,4 +32,4 @@ Use the following library to work with the Azure Event Grid resource provider:
 
 | Nuget Package | Reference |
 |--------------------------------------|---------------------------------------------------------------|
-| [Microsoft.Azure.Management.EventGrid](https://www.nuget.org/packages/Microsoft.Azure.Management.EventGrid)  | [API Reference for Microsoft.Azure.Management.EventGrid](https://docs.microsoft.com/dotnet/api/overview/azure/eventgrid/management?view=azure-dotnet)  |
\ No newline at end of file
+| [Microsoft.Azure.Management.EventGrid](https://www.nuget.org/packages/Microsoft.Azure.Management.EventGrid)  | [API Reference for Microsoft.Azure.Management.EventGrid](https://learn.microsoft.com/dotnet/api/overview/azure/eventgrid/management/management-eventgrid?view=azure-dotnet)  |
```

Comment:
It is intentional that link differs from the pattern for the other two libraries; this is the structure in MS Docs.  I believe the difference is because there is only a T1 management package for Event Grid and the other two have both T1 and T2 packages.